GP-Engine-team/GP_Engine
# Fixed various compile-time and runtime errors and warnings

## Description

** IMPORTANT:** you must first approve [this pull request on GPM](https://github.com/GP-Engine-team/gpm/pull/35) in order to approve this one

This pull request solves several different kinds of warnings at compilation and at runtime.
The solution's project properties have also been cleaned and updated.
PhysXSystem's constructor has been lightly cleaned. The unused part of PhysX has been commented out, and the destructor's content has been uncommented.

## How to test
1. Completely clean the whole solution
2. Build the whole solution
3. Watch the compilation output in Visual Studio to see if there are any errors/warnings left

## IMPORTANT
**Please play the simulation, and check that you don't have any error while shooting**
